fvm/meter/meter_test.go

```diff
@@ -1,7 +1,6 @@
 package meter_test
 
 import (
-	"fmt"
 	"math"
```

Comment:
revert import needed for assertion

```suggestion
	"fmt"
	"math"
```